src/alpha-lab/dense-pose.md

```diff
@@ -1,145 +1,78 @@
 
-## Running in your browser
+Until recently, existing methods for tracking body parts in eye tracking scene video were not very robust or accurate. However, Meta has released DensePose, a new tool that promises to solve many of these problems. As a result, we have decided to incorporate DensePose into our guide. For more information, please refer to [DensePose: Dense Human Pose Estimation In The Wild](https://arxiv.org/abs/1802.00434).
 
-Firstly, from the raw data export, upload **(uncompressed)** one of the subfolders(recording folder) that you're interested in  to Google Drive.
+## Steps
 
-Now for the Google Colab part: [click here to access the notebook](https://colab.research.google.com/drive/1s6mBNAhcnxhJlqxeaQ2IZMk_Ca381p25#forceEdit=true&sandboxMode=true), and carefully follow the instructions there.
+1. Download a [Raw data export]() from your project in Pupil Cloud.
+2. Upload (uncompressed) one of the subfolders (recording folder) that you're interested in to Google Drive from the raw data export.
+3. Access our [Google Colab Notebook](https://colab.research.google.com/drive/1s6mBNAhcnxhJlqxeaQ2IZMk_Ca381p25?usp=sharing) and carefully follow the instructions.
 
 <div class="mb-4" style="display:flex;justify-content:center;">
-  <a href="https://colab.research.google.com/drive/1s6mBNAhcnxhJlqxeaQ2IZMk_Ca381p25#forceEdit=true&sandboxMode=true" target="_blank">
+  <a href="https://colab.research.google.com/drive/1s6mBNAhcnxhJlqxeaQ2IZMk_Ca381p25?usp=sharing" target="_blank">
     <img style="width:180px" src="https://img.shields.io/static/v1?label=&message=Open%20in%20Google%20Colab&color=blue&labelColor=grey&logo=Google%20Colab&logoColor=#F9AB00" alt="colab badge">
   </a>
 </div>
 
-::: tip
-The notebook contains lots of rows and this can seem overwhelming. It needn't be. It's actually really simple. Just click on each of the play buttons...
-:::
-
-## Outputs
-
-After the code is executed, new files will be generated. Check the new DensePoseColab folder. Let's see what's in those files!
-
-#### Video
-
-You've already seen the accompanying video above. There's no mystery – you have a bounding box delimiting each person detected, a blue shaded mask over the body parts, a yellow highlighted body part when it's gazed at, and the typical red circle for the gaze position.
+<div class="mb-4" style="display:flex;justify-content:center;">
+  <v-img class="rounded" :src="require(`../media/alpha-lab/densepose-colab.png`)" width="100%" />
```

Comment:
This colab screenshot isn't doing much to enhance the article IMO. Can we consider omitting it?

Reply:
I will omit it